### PR TITLE
Issue #15214: migrated section 5.2.7 local variable names to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214

### CODE REVIEW

File: src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/InputPatternVariableNameEnhancedInstanceofTestDefault.java [hunk 1]
```diff
@@ -65,17 +70,17 @@ public void t(Object o1, Object o2) {
...
+      {
+        while (!(o1 instanceof String _aa)) {
+          // violation above 'Pattern variable name '_aa' must match pattern'
+          L3:
+          break L3;
+        }
+        while (o1 instanceof String aa_) {
+          // violation above 'Pattern variable name 'aa_' must match pattern'
+          aa_.length();
+        }
```
Comment (Zopsss) [inline src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/InputPatternVariableNameEnhancedInstanceofTestDefault.java:82]: known indentation defect in our config: https://github.com/checkstyle/checkstyle/issues/14294#issuecomment-1904385363

Comment (romani) [reply src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/InputPatternVariableNameEnhancedInstanceofTestDefault.java:82]: Issue is moved into Todo to be stand out of crowd

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**